documentation/docs/adapters/MythicMobsAdapter/entries/action/despawn_mob.mdx

```diff
@@ -0,0 +1,17 @@
+import {ActionsField, EntryField} from "@site/src/components/EntryField";
+
+# Despawn Mob Action
+
+The `Despawn Mob Action` action removes MythicMobs's mob from the world.
+
+## How could this be used?
+
+Hmmm. Clearing the mob through dialogue?
```

Comment:
How are you using this entry? You can describe similar functionality.

The `How can this be used?` section is meant to give new users ideas of how to use an entry. Give ideas of scenarios where this entry could be useful.